[Docs feedback] BD-3487: Clarify columns in CSV file for promotion codes (#8202)
braze-inc/braze-docs: _docs/_user_guide/personalization_and_dynamic_content/promotion_codes.md

### Step 4: Promotion code upload

Braze does not manage code creation or redemption, meaning you must generate your promotion codes to a CSV file and upload them to Braze. Make sure the CSV file follows these guidelines:

> - Has one code per row.
> - Includes a column for the promotion code, which is the actual code you want to apply to the user.

Author: ```suggestion
- Includes a column for promotion codes.
- Has one promotion code per row.
```

> You can use our built-in integration with [Voucherify]({{site.baseurl}}/partners/channel_extensions/loyalty/voucherify/) or [Talon.One]({{site.baseurl}}/partners/channel_extensions/loyalty/talonone/) to create and export promotion codes.

Reviewer: ```suggestion
You can use our built-in integration with [Voucherify]({{site.baseurl}}/partners/message_orchestration/channel_extensions/loyalty/voucherify/) or [Talon.One]({{site.baseurl}}/partners/message_orchestration/channel_extensions/loyalty/talonone/) to create and export promotion codes.
```

Removing redirects
